lib/ansible/modules/files/file.py

```diff
@@ -77,11 +77,12 @@
         "path" file and create symlink to the "src" file in place of it).'
   follow:
     required: false
-    default: "no"
+    default: "yes"
     choices: [ "yes", "no" ]
     version_added: "1.8"
     description:
       - 'This flag indicates that filesystem links, if they exist, should be followed.'
+      - 'Previous to Ansible 2.5, this was C(yes) by default.'
```

Comment:
Shouldn't this read that, previous to 2.5, it was "no" by default?

Reply:
My bad. Will fix it now.